src/openms/source/TRANSFORMATIONS/RAW2PEAK/PeakPickerHiRes.cpp

```diff
@@ -538,7 +538,7 @@ namespace OpenMS
     }
     endProgress();
 
-    OPENMS_LOG_INFO << "Picked spectra by MS-level:\n";
```

Comment:
Is this clearer?

Reply:
Yes because 0 picked spectra sounds like it failed